bananagram-frontend/src/components/Grid/GridControls/GridControls.tsx

```diff
@@ -22,13 +31,32 @@ const ControlButtonsWrapper = styled.div`
 
 export const GridControls: FC<GridControlsProps> = ({
   onZoomIn,
+  onSave,
   onHome,
+  onRestore,
   onZoomOut,
+  offset,
+  dimensions,
+  offsetSaved,
+  dimensionsSaved,
+  initialOffset,
+  initialGridDimensions,
 }) => (
   <GridControlsWrapper>
     <ControlButtonsWrapper>
       <Button onClick={onZoomOut}>-</Button>
-      <Button onClick={onHome}>Home</Button>
+      <Button onClick={onSave}>
+        Save{'('}
```

Comment:
Do we need the `{')'}` here?
Could we just do `Save(`?
Or join it with the next line?
```suggestion
        {`Save(${offset.x}, ${offset.y}@${dimensions.x}x${dimensions.y})`}
```

Reply:
1a110cc done thank you